src/globals.h

```diff
@@ -25,15 +33,17 @@ typedef struct {
   int8_t blescancycle;                 // BLE scan cycle [seconds]
   int8_t blescan;                      // 0=disabled, 1=enabled
   int8_t wifiant;                      // 0=internal, 1=external (for LoPy/LoPy4)
+  int8_t rgblum;                       // RGB Led luminosity (0 100%)
   char version[10];                    // Firmware version
   } configData_t;
 
 extern configData_t cfg;
 extern uint8_t mydata[];
 extern uint64_t uptimecounter;
 extern osjob_t sendjob;
-extern int macnum, blenum, countermode, screensaver, adrmode, lorasf, txpower, rlim, salt;
+extern int countermode, screensaver, adrmode, lorasf, txpower, rlim, salt;
 extern bool joinstate;
+extern std::set<uint32_t> wifis; 

```

Comment:
I downsized the set containers to uint16_t, becauses hashes are 4 byte long. Should be enough, we can count a maximum of 65535 MACs this way. But not sure, if 16bit set gives better performance than 32bit set on ESP32.